harness/determined/common/experimental/determined.py

```diff
@@ -255,3 +255,33 @@ def get_model_labels(self) -> List[str]:
         Get a list of labels used on any models, sorted from most-popular to least-popular.
         """
         return list(bindings.get_GetModelLabels(self._session).labels)
+    
+    def list_clients(self) -> Sequence[oauth2_scim_client.Oauth2ScimCient]:
+        try:
+            oauth2_scim_clients: List[oauth2_scim_client.Oauth2ScimCient] = []
+            clients = api.get(self.master, "oauth2/clients").json()
+            for client in clients: 
+                osc  = oauth2_scim_client.Oauth2ScimCient(name=client["name"],id=client["id"], domain=client["domain"])
+                oauth2_scim_clients.append(osc)
+            return clients
+        except api.errors.NotFoundException:
+            raise EnterpriseOnlyError("API not found: oauth2/clients")
+    
+    def add_client(self, domain: str, name: str) ->  oauth2_scim_client.Oauth2ScimCient:
+        try:
+            client = api.post(
+               self.master,
+                "oauth2/clients",
+                json={"domain": domain, "name": name},
+            ).json()
+            
+            return oauth2_scim_client.Oauth2ScimCient(id = str(client["id"]), secret=str(client["secret"]), domain=domain, name=name)
+
+        except api.errors.NotFoundException:
+            raise EnterpriseOnlyError("API not found: oauth2/clients")
+    
+    def remove_client(self,client_id: str) -> None:
```

Comment:
functions exposed on the Determined object should also be exposed in the det.experimental.client module.  Just follow the same pattern you see in that file.

Reply:
oops yes I forgot about client. I'll do that once we decide whether we really even want these in determined.py